src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin: 0 8px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-field {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input-icon {
+  width: 100%;
+}
+
+.input-icon img {
+  position: absolute;
+}
+
+.icon--big {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon--small {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
Use css variables for colors. 

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)